src/sonic-config-engine/sonic-cfggen

```diff
@@ -57,7 +57,9 @@ def sort_by_port_index(value):
     if not value:
         return
     if isinstance(value, list):
-        value.sort(key = lambda k: int(k[8:]))
+        value.sort(
+            key = lambda k: int(k[8:]) if "BP" not in k else int(k[11:]) + 1024
```

Comment:
it is difficult to understand the logic here.

Reply:
Backend ethernet ports are identified as 'Ethernet-BPxy'. The change extracts the port number for the backend ports. '1024' is just to sort the back end ports after the front end ports, assuming number of front end ports will be < 1024.
'+ 1024' has no functional impact other than for cleaner look in config_db. All backend ports are grouped at the end. This can be removed if that is preferred.